### DISCUSSION TITLE
Adding a Wiki

### DISCUSSION THREAD

**Discussion by kylereddoch** [iOSDevSpace/iosdevspace#12] [Ideas]
I'm proposing adding a wiki (possibly at wiki.iosdev.space) as a central place to house information for both new members and all members to compliment getting started with Mastodon and our instance. It will have things like guides for different apps, setting up their account, info about the instance, tools, etc. 

Here's a good example: https://wiki.dmv.community

I am more than happy to create and maintain this wiki for the instance as the Mod.

**Comment by heyjaywilson** [comment]
There is a plan to make this happen, but I'd like to figure out how to make it in Swift first. Maybe with something like DOCC or even just Publish
